src/components/ebay-notice/README.md

```diff
@@ -34,7 +34,8 @@ Name | Type | Stateful | Required | Description
 `type` | String | No | No | "inline", "section", "window", or "page" (default)
 `status`  | String | No | No | "attention" (default for "page" and "inline"), "confirmation" "information", or "celebration" (page notice only).  The default for "section" type will render with grey background and no icon
 `a11y-heading-text` | String | No | Yes | adding description for the notice for a11y users
-`hidden` | Boolean | Yes | No | whether the widget is hidden or not.
+`hidden` | Boolean | Yes | No | whether the widget is hidden or not
+`icon-hidden` | Boolean | Yes | No | whether the icon is hidden or not
```

Comment:
Sorry, one more request. Rather than a boolean (which often lead us into traps), I prefer the idea making this an enumerate type instead:

`icon` | String | Yes | No | "default" (matches whatever specified by the "type") or "none"

This keeps thing a little more open in terms of allowing different kinds of icon in future if we want. @agliga thoughts?

Reply:
Agreed, I think it is more fluid that way and we might want to add something like right icon or something else. Better to create it more extensible.